src/bio/bio_internal.h

```diff
@@ -15,6 +15,8 @@
 #include <spdk/bdev.h>
 #include <spdk/thread.h>
 
+#include "smd.pb-c.h"
```

Comment:
nit - Anything from this file actually used in this header? I'd lean toward including headers only in files that need them.

Reply:
no, the function that did was added and removed during the PR commits, I will remove in the following PR or if I have to re-push

Reply:
addressing in https://github.com/daos-stack/daos/pull/10768